frontend/components/login-view/login-view.ts

```diff
@@ -31,6 +31,7 @@ export class LoginView extends LitElement implements AfterEnterObserver {
   constructor(){
     super();
     this.onSuccess = () => {
+      localStorage.setItem('loggedIn', String(true));
```

Comment:
Since the value needs to be a string, I'd recommend using '0' and '1' as the possible values. If you use 'true' and 'false' like it is now, you might accidentally try to parse it as a boolean (like you've done in isUserLoggedIn below), but both will be true when cast to boolean.